### PR TITLE
Added a check for VASP start and fixed quote consistency

### PR DESCRIPTION
**Pull request by espenfl**
Here we add a trigger for the stream parser that ends up setting an `INFO` message if VASP was started. However, note that this check tells nothing about the run being successful, meaning complete or not. Only that VASP has been started.

### CODE REVIEW

**Comment by espenfl** [comment]
@JPchico @atztogo @zhubonan Would be sweet if one of you could have a quick look at this. It preps to use this in AiiDA-VASP to set the right exit code when `vasp_output` is not empty (for instance if `mpirun` etc. is not available). Only states that VASP was started, not that it executed cleanly.

**Comment by atztogo** [comment]
@espenfl, I don't see what this PR means.

**Review by JPchico — COMMENTED** [review]
The changes all seem fine, I'm not entirely sure on how it is checking whether or not VASP has started.

**Comment by zhubonan** [comment]
Looks fine to me! One thing that I am not sure is that the notification is emitted when there is no problem, but in other cases the notifications are there for errors. 

This logic is fine, but there is a potential problem when working with `aiida-vasp` - one must make sure `parservasp` is up-to-date otherwise, aiida-vasp (with the checks for the new INFO notification) will think every calculation is not run. This might comes as a surprise if one updates aiida-vasp but did not do so for parsevasp. 

Maybe a better solution is to have a negative match where the notification is emitted when the stream does not detect `vasp.` regex? 

The bits we need to change aiida-vasp is here:
https://github.com/aiida-vasp/aiida-vasp/blob/a0fc60b15be0f3e2c286659b2252ff57e416fa2f/aiida_vasp/parsers/vasp.py#L277-L317

**Comment by espenfl** [comment]
> @espenfl, I don't see what this PR means. How does this couple with external code like aiida?

This is to give the option in `aiida-vasp` to fix the fact that we set an exit code that is not so useful at first for the user in case VASP is never started. There, we now only check if the `vasp_output` is present, which is not enough.

File: parsevasp/stream.yml [hunk 1]
```diff
@@ -7,180 +7,188 @@
 # location: Can be found in which stream (STDOUT/STDERR)
 # recover: If False, always break when this error appears, there is no point in recovering
 
+# A special one to detect if VASP have been started
+started:
```
[the remark this answers is not in the document]
Comment (espenfl) [reply parsevasp/stream.yml:11]: Yes, I agree with this. Good suggestion. I will update it.

**Comment by atztogo** [comment]
> > @espenfl, I don't see what this PR means. How does this couple with external code like aiida?
> 
> This is to give the option in `aiida-vasp` to fix the fact that we set an exit code that is not so useful at first for the user in case VASP is never started. There, we now only check if the `vasp_output` is present, which is not enough.

I see. Thanks for your explanation. By the way, this commit includes refactoring (which could be another PR, too). How did it happen?

**Comment by espenfl** [comment]
@zhubonan Do you mind having another look. Thanks.

**Review by zhubonan — APPROVED** [review]
Looks good to me!

**Pull request merged**